encryption.rst

```diff
@@ -32,6 +28,20 @@ Also, decryption still requires a setuid installation of {Project}.
    Ubuntu 18.04, Debian 10 and CentOS/RHEL 7, but users of older Linux
    versions may have to update.
 
+.. note:: 
+
+   We use ``Privileged Encryption`` to represent the case of building encrypted image,
+   where root user or {Project} suid installation are required. Partition type in SIF image
+   will be shown as ``Encrypted squashfs`` in this case.
```

Comment:
Change this paragraph to
```
We use ``Privileged Encryption`` to represent the case of an encrypted image built by the
root user and decrypted with a suid installation.  The partition type in a SIF image in this
case wil be shown as ``Encrypted squashfs``.
```

Reply:
resolved 
https://github.com/apptainer/apptainer-userdocs/pull/179/commits/61f67f30ed051f03176f77556533a3d5902aefd8